fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommandManager.java

```diff
@@ -49,23 +52,31 @@
  * <h2>Example command</h2>
  * <pre>
  * {@code
- * ClientCommandManager.DISPATCHER.register(
- * 	ClientCommandManager.literal("hello").executes(context -> {
- * 		context.getSource().sendFeedback(new LiteralText("Hello, world!"));
- * 		return 0;
- * 	})
- * );
+ * ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> {
+ * 		dispatcher.register(
+ * 			ClientCommandManager.literal("hello").executes(context -> {
+ * 				context.getSource().sendFeedback(Text.literal("Hello, world!"));
+ * 				return 0;
+ * 			})
+ * 		);
+ * });
  * }
  * </pre>
  */
 @Environment(EnvType.CLIENT)
 public final class ClientCommandManager {
+	private ClientCommandManager() {
+	}
+
 	/**
-	 * The command dispatcher that handles client command registration and execution.
+	 * Gets the active command dispatcher that handles client command registration and execution.
+	 *
+	 * <p>May be null when not connected to a server.</p>
```

Comment:
If client commands do work in integrated environment, this is misleading, since you only “connect” to dedicated servers. “not in a world” might work.